packages/components/src/components/canvas/canvas.tsx

```diff
@@ -63,6 +63,41 @@ export class Canvas {
   private hasFooter: boolean;
   private hasBackground: boolean;
 
+  private sidebarStart: HTMLElement;
+  private sidebarEnd: HTMLElement;
+  private root: HTMLElement;
+  private header: HTMLHeadElement;
+
+  @Watch('sidebarStartOpen')
+  public openChangeHandlerSidebarStart(isOpen: boolean): void {
+    if (this.isMediaQueryS) {
+      if (isOpen) {
+        this.root.ontransitionend = (): void => {
+          this.sidebarStart.focus({ preventScroll: true });
+        };
+      } else {
+        this.root.ontransitionend = (): void => {
+          this.header.focus({ preventScroll: true });
+        };
+      }
+    }
```

Comment:
Would this work too? Maybe a little bit less repetitive.
```suggestion
      this.root.ontransitionend = (): void => {
         this[isOpen ? 'sidebarStart' : 'header'].focus({ preventScroll: true });
      };
    }
```